get_token.py

```diff
@@ -0,0 +1,4 @@
+from methods.shared.common import get_auth
+
+if __name__ == "__main__":
+    print(get_auth("dispenser", "INT"))
```

Comment:
Do the args need hard-coding here? They can be passed-in when invoking the script, or made constants in the script itself if they are not to be changed.

Reply:
I created a new function for the internal-dev token as I didn't want to mess too much with getting the INT token. Tell me if you agree with the change.